app/services/foreman/plugin.rb

```diff
@@ -67,6 +67,16 @@ def register(id, &block)
         registered_plugins[id] = plugin
       end
 
+
+      def modify_backtrace_cleaner
+        cleaner = Rails.backtrace_cleaner
+        cleaner.remove_silencers!
+        cleaner.add_silencer do |line|
+          line !~ Rails::BacktraceCleaner::APP_DIRS_PATTERN &&
+              !(all.any? { |plugin| line.include?(plugin.name) })
```

Comment:
unfortunatelly, the rails add the filters to the backtrace, to replace the full path with the more readable prefix, containing just the name and version instead of full path (https://github.com/rails/rails/blob/master/railties/lib/rails/backtrace_cleaner.rb#L28), so the lines we match look like this:

```
foreman_setup (2.1.1) app/controllers/foreman_setup/provisioners_controller.rb:175:in `block in step4_update'
```